### ISSUE TITLE
resolve function in FHIRPATH expression

### ISSUE THREAD

**Issue by dipanjanmukherjeeibm** [IBM/FHIR#2296]
**Is your feature request related to a problem? Please describe.**
resolve() function is a function in FHIR that allows to reference elements  across FHIR resources. The resolve() function currently doesn't work in IBM FHIR server.

**Describe the solution you'd like**
resolve() function is a function in FHIR that allows to reference elements  across FHIR resources. The resolve() function is integral to build business rules for FHIR validation requireing cross-resource validation. The documentation for resolve () function in FHIR server states:
 ```
 /**
     * For each item in the collection, if it is a string that is a uri (or canonical or url), locate the target of the
     * reference, and add it to the resulting collection. If the item does not resolve to a resource, the item is ignored
     * and nothing is added to the output collection. The items in the collection may also represent a Reference, in which
     * case the Reference.reference is resolved.
     *
     * <p>This method creates a resource node that is a placeholder for the actual resource, thus allowing for the FHIRPath
     * evaluator to perform type checking on the result of the resolve function. For example:
     *
     * <pre>Observation.subject.where(resolve() is Patient)</pre>
     *
     * <p>If the resource type cannot be inferred from the reference URL or type, then {@code FHIR_UNKNOWN_RESOURCE_TYPE} is used
     * so that we index these references by default.
     *
     * @param evaluationContext
     *     the evaluation environment
     * @param context
     *     the current evaluation context
     * @param arguments
     *     the arguments for this function
     * @return
     *     the result of the function applied to the context and arguments
     */
```
**Describe alternatives you've considered**
example of a FHIR path expression with resolve function that would work in the IBM FHIR Server on coverage profile
`period.end <= beneficiary.resolve().deceased as deceasedDateTime`

**Acceptance Criteria**
Any FHIR PATH expression with resolve() function should work when the profile on the IG is invoked by IBM FHIR Server during validation.

**Additional context**

**Comment by lmsurpre** [comment]
@dipanjanmukherjeeibm this feature was added and released in IBM FHIR Server 4.8.1.

To enable it, you must set `fhirServer/core/serverResolveFunctionEnabled` to `true` in your fhir-server-config.json

Please give it a try and confirm that this addresses your request.  If we don't hear back in a day or so, I will plan to close this one out.

**Comment by dipanjanmukherjeeibm** [comment]
I have tried to add the above in the config file and the problem is now resolved. I am happy for the dev team  to close this request as resolved
